suites/quincy/rados/tier-2_rados_test-pg-dups-trimming.yaml

```diff
@@ -24,7 +25,7 @@ tests:
                 # using 6.1 rc candidate
                 custom_image: "registry-proxy.engineering.redhat.com/rh-osbs/rhceph:6-177"
                 custom_repo: "http://download-node-02.eng.bos.redhat.com/rhel-9/rel-eng/RHCEPH-6/RHCEPH-6.1-RHEL-9-20230602.0/compose/Tools/x86_64/os/"
```

Comment:
As we are not following an upgrade path and PG Dups trimming verification is possible in a greenfield deployment, is it necessary to identify custom image of RHCS 6.1 rc build?

Reply:
Please disregard the above comment, it seems to be my misunderstanding, I did find the code to upgrade the cluster in test `tests/rados/test_pg_dups_trimming.py`